.github/workflows/pr-helper.yml

```diff
@@ -0,0 +1,25 @@
+name: Semantic PR helper
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize, labeled]
+
+permissions:
+  pull-requests: write # Write access needed to create a comment.
+
+jobs:
+  pr-help:
+    name: Semantic PR helper
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to push code
+      issues: write # to make comments on issues
+      pull-requests: write # to make comments on PR
```

Comment:
```suggestion
```

Permissions are already defined at the top level

Reply:
Oops, sorry, that was a bit of copy paste.